requirements.txt

```diff
@@ -8,6 +8,7 @@ Jinja2>=2.7.3
 Sphinx>=1.2.3
 docopt>=0.6.2
 simplejson>=3.8.1
+urllib3>=1.9
```

Comment:
Strange I have not encountered an error that request this lib.
Where did you see that ?